052-SAPAppModernization/Coach/Solution-01.md

```diff
@@ -0,0 +1,33 @@
+# Challenge 01 - Rapid SAP deployment - Coach's Guide 
+
+[< Previous Challenge](./Solution-00.md) - **[Home](./README.md)** - [Next Solution >](./Solution-02.md)
+
+# Notes & Guidance
+
+The most likely things to go wrong here are
+
+1. The user has no SAP subscription or licence and has no S-number. They simply will need to be registered with SAP within their organization to get this.
+2. They have not linked their Cloud Appliance Library account to their Azure subscription.
+3. They have not configured a service principal with the appropriate rights and added it to allow the SAP Cloud Appliance library to deploy into their subscription.
+
+## Description
+
+During the exercise, participants will be able to provision a landscape into Azure for SAP environment and then build a fresh SAP S4H system by deploying SAP HANA as a claoud appliance. This system will be licenced for one month's trial purposes.
+
+## Prerequisite
+
+A valid Azure subscription which can create azure resources (computed D-series, E-series, storage, Network..).
```

Comment:
Capitalize Azure. What is computed D-series? Should it be compute? Change network to lower case